CMakeLists.txt

```diff
@@ -26,7 +29,11 @@ target_include_directories(libzippp
     $<INSTALL_INTERFACE:include/libzippp>
 )
 set_target_properties(libzippp PROPERTIES PREFIX "") # Avoid duplicate "lib" prefix
-target_link_libraries(libzippp PRIVATE libzip::libzip)
+if(UNIX OR APPLE)
+  target_link_libraries(libzippp PRIVATE libzip::libzip OpenSSL::SSL OpenSSL::Crypto)
+else()
+  target_link_libraries(libzippp PRIVATE libzip::libzip Crypt32.lib)
+endif()
```

Comment:
With that, we are enforcing the library users to include also a cryptographic library which, in most of the case is useless. I think it would be valuable to make it optional (by using a `#define`) and update the documentation. WDYT ?

Reply:
I think so too. I will implement that as well.